### ISSUE TITLE
Welcome to StatsCAN screen

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#314]
When using accessibility controls:

Welcome to StatsCAN should be labelled as heading

**Comment by ForumG-SC** [comment]
Done
